<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addvehicle</class>
 <widget class="QWidget" name="addvehicle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Vehicle</string>
   </property>
  </widget>
  <widget class="QLabel" name="makeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vehicle Make: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="makeLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vehicle Model: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vehicle Priority</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lowRadioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="medRadioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="highRadioButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>High</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>makeLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>addvehicle</receiver>
   <slot>makeEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modelLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>addvehicle</receiver>
   <slot>modelEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>addvehicle</receiver>
   <slot>lowChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>medRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>addvehicle</receiver>
   <slot>medChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>addvehicle</receiver>
   <slot>highChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>addvehicle</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>makeEdited()</slot>
  <slot>modelEdited()</slot>
  <slot>lowChecked()</slot>
  <slot>medChecked()</slot>
  <slot>highChecked()</slot>
  <slot>add()</slot>
 </slots>
</ui>
